src/components/productGallery/ProductGallery.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import "./ProductGallery.css";
+
+const ProductGallery = ({
+  bigImage,
+  smallTopLeftImage,
+  smallTopRightImage,
+  smallBottomLeftImage,
+  smallBottomRightImage,
+}) => {
+  const [showPopupProductGallery, setShowPopupProductGallery] = useState(true);
```

Comment:
The popup should initially be hidden, so you need to set it to `false`. If it's set to `true`, the popup will be displayed immediately when the component renders.

Reply:
Thank you for the point. I will change it.